src/rar2john.c

```diff
@@ -256,7 +267,6 @@ static void process_file(const char *archive_name)
 			MEM_FREE(gecos);
 			if (process_file5(archive_name))
 				return;
-			fprintf(stderr, "! %s: Not a RAR file\n", archive_name);
```

Comment:
That same message would now be printed from `process_file5()` before we got here.

Reply:
> That same message would now be printed from `process_file5()` before we got here.

I see now that probably goes for line 258 in new code as well. Will test more and amend this.